drivers/lte_link_control/Kconfig

```diff
@@ -25,6 +25,14 @@ config LTE_LOCK_BANDS
 	help
 		Enable LTE band locks for bands 3, 4, 13 and 20.
```

Comment:
This help string seems no longer valid with `LTE_LOCK_BAND_MASK` added.

Reply:
Changed the strings to be more correct and descriptive